Lib/test/test_ntpath.py

```diff
@@ -422,15 +422,20 @@ def test_ismount(self):
             self.assertTrue(ntpath.ismount(b"\\\\localhost\\c$"))
             self.assertTrue(ntpath.ismount(b"\\\\localhost\\c$\\"))
 
+    def assertEqualCI(self, s1, s2):
+        """Assert that two strings are equal ignoring case differences
```

Comment:
PEP 8 would want this as:
```python
"""Assert that two strings are equal, ignoring case differences."""
```

IOW one line, ending in a period.

Reply:
Done